lib/connect-dynamodb.js

```diff
@@ -106,7 +106,7 @@ module.exports = function (connect) {
     DynamoDBStore.prototype.get = function (sid, fn) {
 
         sid = this.prefix + sid;
-        var now = +new Date;
+        var now = Math.floor(Date.now() / 1000);
```

Comment:
I think any existing expired records would be returned normally because `now >= result.Item.expires` would be false until the record was updated. I think anyone who wants to use the update can purge out old records and then apply the update. After it is life they might want to make sure everything is in seconds and not milliseconds.